libs/design-system/src/atoms/typography/baseText.tsx

```diff
@@ -0,0 +1,32 @@
+import { cn } from '../../shared/classnames';
+
+type ParagraphProps = {
+  type?: 'sm' | 'base' | 'lg';
+  alignment?: 'left' | 'center' | 'right';
+  className?: string;
+  children: React.ReactNode;
+};
+
+const fontSizeMapping = {
+  sm: 'text-3.5',
+  base: 'text-4',
+  lg: 'text-5',
+};
+
+const aligmentMapping = {
+  left: 'text-left',
+  center: 'text-center',
+  right: 'text-right',
+};
+
+const Paragraph = ({ type = 'base', alignment = 'left', className, children }: ParagraphProps) => {
+  return (
+    <p className={cn('antialiased', 'text-gray', fontSizeMapping[type], aligmentMapping[alignment], className)}>
```

Comment:
`alignmentMapping`

You can have a look at the VSCode extension Code Spell Checker